### PR TITLE
8267582: BasicLookAndFeel should not call getComponentPopupMenu twice to get a popup menu

### PR DESCRIPTION
**Pull request by SWinxy**
Please review this small change to cache a component's popup menu in BLAF.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8267582](https://bugs.openjdk.org/browse/JDK-8267582): BasicLookAndFeel should not call getComponentPopupMenu twice to get a popup menu

### Reviewers
 * [Prasanta Sadhukhan](https://openjdk.org/census#psadhukhan) (@prsadhuk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/12017/head:pull/12017` \
`$ git checkout pull/12017`

Update a local copy of the PR: \
`$ git checkout pull/12017` \
`$ git pull https://git.openjdk.org/jdk pull/12017/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 12017`

View PR using the GUI difftool: \
`$ git pr show -t 12017`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/12017.diff">https://git.openjdk.org/jdk/pull/12017.diff</a>

</details>

### CODE REVIEW

File: src/java.desktop/share/classes/javax/swing/plaf/basic/BasicLookAndFeel.java [hunk 1]
```diff
@@ -25,60 +25,31 @@
...
-import javax.swing.JTextField;
-import javax.swing.DefaultListCellRenderer;
-import javax.swing.FocusManager;
-import javax.swing.LayoutFocusTraversalPolicy;
-import javax.swing.SwingUtilities;
-import javax.swing.MenuSelectionManager;
-import javax.swing.MenuElement;
-import javax.swing.border.*;
+import javax.sound.sampled.*;
+import javax.swing.*;
```
Comment (prsadhuk) [inline src/java.desktop/share/classes/javax/swing/plaf/basic/BasicLookAndFeel.java:35]: Fix looks ok.
But , please replace the wildcard imports with specific ones

Comment (SWinxy) [reply src/java.desktop/share/classes/javax/swing/plaf/basic/BasicLookAndFeel.java]: Thank you!

**Review by prsadhuk — APPROVED** [review]

**Comment by SWinxy** [comment]
/integrate

Thank you thank you, Prasanta.

**Comment by SWinxy** [comment]
@TheShermanTanker you might be needed here!

**Comment by TheShermanTanker** [comment]
Good Work buddy :)

**Pull request closed without merging**